website/src/data/gallery.json

```diff
@@ -198,5 +198,12 @@
     "description": "A multimodal webbrowsing agent that autonomously browses the web.",
     "image": "webagent.jpg",
     "tags": ["tools", "app"]
+  },
+  {
+    "title": "Create Issues from Code Commits - using Autogen",
+    "link": "https://blog.composio.dev/automating-task-creation-with-openai-assistant-from-code-comments-to-linear-issues/",
+    "description": "Discover how to automatically create Linear issues from GitHub commits using Autogen Agents.",
```

Comment:
```suggestion
    "description": "Automatically creating Linear issues from GitHub commits using AutoGen Agents.",
```

Reply:
Can you expand a bit more on "Linear issues" in the description.

Reply:
Hey Eric, Just updated in this [commit](https://github.com/microsoft/autogen/pull/2237/commits/9f48789a51790957431b6900705c761986c26c8e). Please review it.